Realm/RLMArray.mm

```diff
@@ -46,6 +46,14 @@ @implementation RLMArray {
     NSMutableArray *_backingArray;
 }
 
++ (void)load
```

Comment:
`+initialize` would be preferable to `+load` since it's sufficient to run this the first time an instance is constructed, rather than as soon as the image is loaded.